src/main/plugins/application-search-plugin/application-icon-service.ts

```diff
@@ -46,7 +46,8 @@ export class ApplicationIconService {
                                 this.logger.debug(`Successfully generated ${applicationsToGenerateIcons.length} icons`);
                                 resolve();
                             })
-                            .catch((err) => reject(err));
+                            .catch(
+                                (err) => reject(err));
```

Comment:
Do you have `prettier` installed and configured as formatter in your editor? Please revert this change as the formatting was correct according to `prettier`'s configuration.